src/measure_theory/measure/hausdorff.lean

```diff
@@ -597,4 +666,110 @@ by rw [sUnion_eq_bUnion, dimH_bUnion hS]
 @[simp] lemma dimH_union (s t : set X) : dimH (s ∪ t) = max (dimH s) (dimH t) :=
 by rw [union_eq_Union, dimH_Union, supr_bool_eq, cond, cond, ennreal.sup_eq_max]
 
+/-!
+### Hausdorff measure and Lebesgue measure
+-/
+
+/-- In the space `ι → ℝ`, Hausdorff measure coincides exactly with Lebesgue measure. -/
+theorem hausdorff_measure_pi_real {ι : Type*} [fintype ι] [nonempty ι] :
+  (μH[fintype.card ι] : measure (ι → ℝ)) = volume :=
```

Comment:
Maybe phrase this for `(n : ℕ) [fact (fintype.card ι) = n] .... μH[n]`, because in the use case `ι = fin n`, I believe `fintype.card (fin n)` is not defeq to `n`.

Reply:
I don't think that we need this. It's easy to add a version for `fin n` with proof `by rw [← fintype.card_fin, hausdorff_measure_pi_real]`.